tests/ops/service/masking/strategy/test_masking_strategy_aes_encrypt.py

```diff
@@ -49,7 +51,7 @@ def test_mask_all_aes_modes(mock_encrypt: Mock):
 
 def cache_secrets() -> None:
     secret_key = MaskingSecretCache[bytes](
-        secret=b"\x94Y\xa8Z",
+        secret=b"y\xc5I\xd4\x92\xf6G\t\x80\xb1$\x06\x19t/\xc4",
```

Comment:
Original AESGCM key in test needed to be 128, 192, or 256 bits - this came up when I tried to use it below intest_mask_arguments_date